taxonomies/ingredients.txt

```diff
@@ -44517,7 +44517,7 @@ pl:Burak
 ru:Свёкла
 sk:cvikly
 sv:rödbeta, rödbetor, rödbets
-wikidata:en:Q165437
+wikidata:en:Q99548274
```

Comment:
Although I understand that squashing commits is recommended in the contributing guide, I thought that for review purposes it might make sense to keep individual data edits separate (so that they're explainable, reviewable and revertible individually).

Reply:
I would say it's up to you to group sensible commit in a single PR and avoid a PR mixing a lot of thing.

It will also help merging PR faster, as if you mix multiple subject, and one is either controversial or problematic, it will block all the other.